tests/test_intercept.js

```diff
@@ -178,15 +178,17 @@ test('reply takes a callback for status code', async t => {
 test('reply should throw on error on the callback', t => {
   let dataCalled = false
 
-  const scope = nock('http://www.google.com')
+  const scope = nock('http://example.com')
```

Comment:
I wonder if we should use non-existing domains to make clear that we don’t intend to send requests to these domains. Something like `http://test.nock` maybe?

Reply:
Sure, that makes a lot of sense. The principle I'd been taking here is trying to remove specific domain names when the test doesn't care.

`test.nock` makes me wonder if it's being treated specially by the code in some way. [It looks like `.test` is intended for this](https://en.wikipedia.org/wiki/.test), so how about using `example.test` to make it clear that this is an arbitrary hostname?

Reply:
`example.test` is great 👍